src/main/java/net/snowflake/ingest/streaming/internal/DropChannelRequestInternal.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.snowflake.ingest.streaming.DropChannelRequest;
+import net.snowflake.ingest.utils.Utils;
+
+/** Class used to serialize the {@link DropChannelRequest} */
+public class DropChannelRequestInternal implements StreamingIngestRequest {
```

Comment:
nit: just DropChannelRequest is enough? why the internal suffix?

Reply:
I see there is already a DropChannelRequest class, curious why we need two classes for the same thing?

Reply:
nvm, i see DropChannelRequest is a publicly exposed class thus we need this internal class.